rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/StatusController.java

```diff
@@ -36,7 +36,7 @@
  */
 @Api(tags = {"Heartbeat", "Version info", "Status"})
 @RestController
-@RequestMapping(path = "/rest/openehr/v1", produces = {MediaType.APPLICATION_XML_VALUE, MediaType.APPLICATION_JSON_VALUE})
+@RequestMapping(path = "/rest", produces = {MediaType.APPLICATION_XML_VALUE, MediaType.APPLICATION_JSON_VALUE})
```

Comment:
I'm not sure, but would it make sense to also move the file out of the `.../rest/openehr/...` scope now?

Reply:
Fixed